src/Sql/dbo/Stored Procedures/Organization_Create.sql

```diff
@@ -168,7 +175,9 @@ BEGIN
         @MaxAutoscaleSmSeats,
         @MaxAutoscaleSmServiceAccounts,
         @SecretsManagerBeta,
-        @LimitCollectionCreationDeletion,
+        COALESCE(@LimitCollectionCreation, @LimitCollectionDeletion, 0), -- Deprecated https://bitwarden.atlassian.net/browse/PM-10863)
```

Comment:
Thinking this through to make sure I understand:

* combination of current code + new db: `@LimitCollectionCreation` and `@LimitCollectionDeletion` are NULL, but they both take the value of `LimitCollectionCreationDeletion` due to the `COALESCE` above. This line ends up writing the current value correctly, even though it doesn't reference `@LimitCollectionCreationDeletion` directly. (And we also set the new properties accordingly.)

* combination of new code + new db: `@LimitCollectionCreationDeletion` will effectively continue to be updated based on the value of `@LimitCollectionCreation`. It won't be used in the new code, so it's only being updated in case there's a rollback scenario. In that case it's not really clear which value it should track given that it doesn't map to either perfectly, but I guess that's still better than it being `NULL`ed out by default.

I think this is OK but please correct me if I'm missing anything. I think this is worth a regression sweep from QA after merging to `main` in any case just to make sure.

Reply:
💯

> In that case it's not really clear which value it should track 

The way it is written now this scenario would result in `LimitCollectionCreationDeletion` being "rolled back" to the value of `LimitCollectionCreation` specifically since it is the first written conditional in the `COALESCE` statement.